virttest/utils_test/__init__.py

```diff
@@ -2038,7 +2038,7 @@ def install(self):
                                     self.work_path)
         self.make_cmds = "cd %s;%s" % (install_path, self.make_cmds)
         logging.info('make and install %s', self.stress_type)
-        status, output = self.cmd_status_output(self.make_cmds, timeout=600)
+        status, output = self.cmd_status_output(self.make_cmds, timeout=self.stress_shell_timeout)
```

Comment:
This could cause failure sometimes, as the make commands are expected to take time more than a minute, request to have separate timeouts for,
1.  `cmd_status()` -> as it wait for the shell to return and 
2. `wait_for()` -> it wait for `True` to be returned by the function passed

Reply:
@balamuruhans I have separated wait_for timeout and shell timeout :+1: